jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -192,6 +194,24 @@ function with_retries() {
   return "$rc"
 }
 
+function check_retry_params() {
+  local rc \
+    cmd=("$@") \
+    retry_interval=$2 \
+    timeout_seconds=$3 \
+
+  if ! [[ "$retry_interval" =~ ^[0-9]+$ ]] || ! [[ "$timeout_seconds" =~ ^[0-9]+$ ]]; then
```

Comment:
Check each parameter individually and report only those which are invalid, or even the first invalid only.

Reply:
I'll do as suggested